<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="programpaths">
      <attribute name="title">
       <string>Programpaths</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="3">
        <widget class="QLineEdit" name="inputSudoOptions"/>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="inputSudo"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>sudo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4" rowspan="2">
        <widget class="QPushButton" name="buttonPing6">
         <property name="text">
          <string>Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="inputPingOptions"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="inputPing6Options"/>
       </item>
       <item row="4" column="3">
        <widget class="QLineEdit" name="inputWhoisOptions"/>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="inputNmap"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Ping6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="inputTracerouteOptions"/>
       </item>
       <item row="5" column="3">
        <widget class="QLineEdit" name="inputNmapOptions"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>NMap</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QPushButton" name="buttonSudo">
         <property name="text">
          <string>Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Whois</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="buttonNmap">
         <property name="text">
          <string>Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="buttonPing">
         <property name="text">
          <string>Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QPushButton" name="buttonWhois">
         <property name="text">
          <string>Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Traceroute</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="buttonTraceroute">
         <property name="text">
          <string>Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="inputPing6"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="inputPing"/>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="inputWhois"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Ping</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="inputTraceroute"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sshOptions">
      <attribute name="title">
       <string>SSH Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Program</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="inputSsh"/>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonSsh">
         <property name="text">
          <string>Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" rowspan="3" colspan="3">
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Entfernen</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Hinzufügen</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Shell Options</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>Terminal</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>User</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Command Options</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLineEdit" name="inputSshOptions"/>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="digOptions">
      <attribute name="title">
       <string>dig Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="1">
        <widget class="QLineEdit" name="inputDig"/>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="terminalDig">
         <property name="text">
          <string>Terminal</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="buttonDig">
         <property name="text">
          <string>Durchsuchen</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Program</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" rowspan="3" colspan="3">
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Hinzufügen</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Entfernen</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QLineEdit" name="inputDigOptions"/>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Server</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Options</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="MacToSwitchPortOptions">
      <attribute name="title">
       <string>NMC</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLineEdit" name="nmcUrl"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="nmcPassword">
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>URL</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="nmcUser"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>User</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="systemNetworkConfig">
      <attribute name="title">
       <string>Network Configuration</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTextEdit" name="networkconfigOutput"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>inputTraceroute</tabstop>
  <tabstop>inputTracerouteOptions</tabstop>
  <tabstop>buttonTraceroute</tabstop>
  <tabstop>inputPing</tabstop>
  <tabstop>inputPingOptions</tabstop>
  <tabstop>buttonPing</tabstop>
  <tabstop>inputPing6</tabstop>
  <tabstop>inputPing6Options</tabstop>
  <tabstop>buttonPing6</tabstop>
  <tabstop>inputWhois</tabstop>
  <tabstop>inputWhoisOptions</tabstop>
  <tabstop>buttonWhois</tabstop>
  <tabstop>inputNmap</tabstop>
  <tabstop>inputNmapOptions</tabstop>
  <tabstop>buttonNmap</tabstop>
  <tabstop>inputSudo</tabstop>
  <tabstop>inputSudoOptions</tabstop>
  <tabstop>buttonSudo</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>inputSshOptions</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>inputDig</tabstop>
  <tabstop>terminalDig</tabstop>
  <tabstop>buttonDig</tabstop>
  <tabstop>listWidget_2</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>inputDigOptions</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>nmcUrl</tabstop>
  <tabstop>nmcPassword</tabstop>
  <tabstop>nmcUser</tabstop>
  <tabstop>networkconfigOutput</tabstop>
  <tabstop>buttonSsh</tabstop>
  <tabstop>inputSsh</tabstop>
  <tabstop>listWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
